### PR TITLE
deps: update statuses to v2.0.1

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

**Comment by dead-horse** [comment]
https://github.com/jshttp/statuses/blob/master/HISTORY.md#200--2020-04-19

looks like this upgrade contains incompatible changes

**Comment by 3imed-jaberi** [comment]
Kindly, can you tell me which change is incompatible and I will see what I can do.

File: test/application/respond.js [hunk 1]
```diff
@@ -374,7 +374,7 @@ describe('app.respond', () => {
     describe('with custom status=700', () => {
       it('should respond with the associated status message', async() => {
         const app = new Koa();
-        statuses['700'] = 'custom status';
```
Comment (dead-horse) [inline test/application/respond.js:377]: this is a breaking change for users who want to customize status code

Comment (dead-horse) [reply test/application/respond.js:377]: IMO we need to provide a more controllable way to customize status code.

Comment (3imed-jaberi) [reply test/application/respond.js:377]: > this is a breaking change for users who want to customize status code

According to what I understand form you, we should leave this PR to Koa@3  !!

Comment (3imed-jaberi) [reply test/application/respond.js:377]: > IMO we need to provide a more controllable way to customize status code.

Perhaps we can provide a method to store, set, cache.. custom messages for custom codes.

I think this approach is simple and clean.

**Review by 3imed-jaberi — COMMENTED** [review]
Updated to latest version v2.0.1 🚀!

**Review by 3imed-jaberi — COMMENTED** [review]
@jonathanong @dougwilson @dead-horse @fengmk2 
Hello guys, take a look if this is considered as a breaking change you can create a PR for koa v3.x where I can compare it or make a label for v3.x ...!

**Review by fengmk2 — APPROVED** [review]

**Comment by jonathanong** [comment]
merged here: https://github.com/koajs/koa/commit/0d9f062aa840c54b5c20911dab6ce62128d3b089 (sorry, you're not the author of this commit, not sure how to do that. I had to rebase)

**Pull request closed without merging**